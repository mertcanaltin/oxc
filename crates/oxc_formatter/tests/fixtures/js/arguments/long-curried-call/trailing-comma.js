Event.debounce(
  call(A, B),
  DDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDD
)(() => { });
