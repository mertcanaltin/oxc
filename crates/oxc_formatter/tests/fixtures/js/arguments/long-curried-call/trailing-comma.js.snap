---
source: crates/oxc_formatter/tests/fixtures/mod.rs
---
==================== Input ====================
Event.debounce(
  call(A, B),
  DDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDD
)(() => { });

==================== Output ====================
-------------------------
{ trailingComma: "none" }
-------------------------
Event.debounce(
  call(A, B),
  DDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDD
)(() => {});

------------------------
{ trailingComma: "es5" }
------------------------
Event.debounce(
  call(A, B),
  DDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDD
)(() => {});

------------------------
{ trailingComma: "all" }
------------------------
Event.debounce(
  call(A, B),
  DDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDD,
)(() => {});

===================== End =====================
